### PR TITLE
add play button

### PR DESCRIPTION
**Pull request by steve02081504**
图片

### CODE REVIEW

**Comment by Zichqec** [comment]
Would it be possible to put the run button next to the "記述例" text instead? 🤔 That way it doesn't interfere with normal copy and pasting of the scripts. Like this:

image

**Review by Zichqec — COMMENTED** [review]
May I suggest calling the event "OnUkadocScriptExample" instead of "OnUkaDocScriptExample"? 🤔 kind of a pedantic change, but I don't think I've seen ukadoc stylized as "UkaDoc" before, and writing it as "Ukadoc" in the event name makes it seem more like a singular word/name

File: manual/sakura_player.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	// 为其增加一个按钮，点击后执行SakuraScript
+	sakuraScriptCodes.forEach(code => {
+		const button = createExecutionButton(code.textContent);
+		code.parentElement.insertBefore(button, code);
+	});
+}
+
+function createExecutionButton(script) {
+	const button = document.createElement("button");
+	button.textContent = "执行";
```
Comment (nikolat) [inline manual/sakura_player.js:24]: Please change to "実行" in Japanese.

File: manual/sakura_player.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	// 为按钮添加悬浮提示
+	button.title = "执行SakuraScript";
```
Comment (nikolat) [inline manual/sakura_player.js:26]: Please change to "SakuraScriptを実行" in Japanese.

**Review by nikolat — COMMENTED** [review]
I would like to see the document standardized into Japanese.

File: manual/sakura_player.js (hunk 1 above)
Comment (steve02081504) [reply manual/sakura_player.js]: fixed at https://github.com/ukatech/ukadoc/commit/9ac8cc660cda3cd8f29ba6aef644faa84fa4b2f4

File: manual/sakura_player.js (hunk 2 above)
Comment (steve02081504) [reply manual/sakura_player.js]: fixed at https://github.com/ukatech/ukadoc/commit/9ac8cc660cda3cd8f29ba6aef644faa84fa4b2f4

**Comment by steve02081504** [comment]
@Zichqec @nikolat all fixed at https://github.com/ukatech/ukadoc/commit/9ac8cc660cda3cd8f29ba6aef644faa84fa4b2f4!

**Review by nikolat — APPROVED** [review]
LGTM👍

**Pull request merged**